rules/integrations/google_workspace/defense_evasion_application_removed_from_blocklist_in_google_workspace.toml

```
@@ -26,7 +26,51 @@ interval = "10m"
 language = "kuery"
 license = "Elastic License v2"
 name = "Application Removed from Blocklist in Google Workspace"
-note = """## Setup
+note = """## Triage and analysis
+
+### Investigating Application Removed from Blocklist in Google Workspace
+
+Google Workspace Marketplace is an online store for free and paid web applications that work with Google Workspace services and with third party software. Listed applications are based on Google APIs or on Google Apps Script and created by both Google and third party developers.
```

Additional suggested edits:

```suggestion
Google Workspace Marketplace is an online store for free and paid web applications that work with Google Workspace services and third-party software. Listed applications are based on Google APIs or the Google Apps Script and are created by Google and third-party developers.
```